### ISSUE TITLE
Error(webrtcvideoengine.cc:1414): webrtc: (voe_audio_processing_impl.cc:1001): virtual int webrtc::VoEAudioProcessingImpl::SetTypingDetectionStatus(bool): not supported

### ISSUE THREAD

**Issue by kenneththorman** [kenneththorman/appspotdemo-mono#4]
Error(webrtcvideoengine.cc:1414): webrtc: (voe_audio_processing_impl.cc:1001): virtual int webrtc::VoEAudioProcessingImpl::SetTypingDetectionStatus(bool): not supported
Warning(webrtcvoiceengine.cc:828): SetTypingDetectionStatus(0) failed, err=8003

**Comment by coreyauger** [comment]
can you please share what solved this for you?

**Comment by kenneththorman** [comment]
Hi Corey

I am not sure off the bat, since it is so long ago. It was not  directly related to only one commit I am afraid. Let me investigate and get back to you.

Regards
Kenneth

**Comment by kenneththorman** [comment]
Actually come to think about this I think this was related to missing permissions for the android app which in turn prevented the signaling to run as expected.

I am pretty sure it was related to the missing app permissions which I changed in this commit
https://github.com/kenneththorman/appspotdemo-mono/commit/69e561106e61ca6a81dd14c50b05c0c90bdcd266

It is so long time ago and I was in a repository/project start up phase with not such a structured approach to all bugs I found so not everything was documented and committed using issues, the project is still not stable enough that I really use issues to track progress it is more adhoc. I am slowly getting there though.

Hope this helps
Regards
Kenneth
